osu.Game.Rulesets.Catch.Tests/TestSceneCatchPlayerLegacySkin.cs

```diff
@@ -1,14 +1,39 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Linq;
 using NUnit.Framework;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Screens;
+using osu.Framework.Testing;
+using osu.Game.Screens.Play.HUD;
+using osu.Game.Skinning;
 using osu.Game.Tests.Visual;
+using osuTK;
 
 namespace osu.Game.Rulesets.Catch.Tests
 {
     [TestFixture]
     public class TestSceneCatchPlayerLegacySkin : LegacySkinPlayerTestScene
     {
         protected override Ruleset CreatePlayerRuleset() => new CatchRuleset();
+
+        [Test]
+        public void TestLegacyHUDComboCounterHidden([Values] bool withModifiedSkin)
+        {
+            if (withModifiedSkin)
+            {
+                AddStep("change component scale", () => Player.ChildrenOfType<LegacyScoreCounter>().First().Scale = new Vector2(2f));
```

Comment:
Hmm, I've made sure to not save the skin to database so that this doesn't pollute things, but yeah, there's the case where the skin manager is still used in other tests of the same test scene. A `ResetDrawableTarget` will do.